### PR TITLE
Create question, answer and comment models

### PR DESCRIPTION
**Pull request by Karthikeyantestpress**

### CODE REVIEW

File: question/models/comment.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    description = models.TextField()
+    commented_by = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name="question_comment_creator"
+    )
+
+    def __str__(self):
+        return f"Commented by {self.commented_by} on {self.question}"
+
+
+class CommentAnswer(TimeStampedModel):
```
Comment (bharathwaaj) [inline question/models/comment.py]:
```suggestion
class AnswerComment(TimeStampedModel):
```

File: question/models/comment.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+from django.contrib.auth.models import User
+from django.db import models
+from model_utils.models import TimeStampedModel
+
+from .answer import Answer
+from .question import Question
+
+
+class CommentQuestion(TimeStampedModel):
```
Comment (bharathwaaj) [inline question/models/comment.py]:
```suggestion
class QuestionComment(TimeStampedModel):
```

**Review by bharathwaaj — CHANGES_REQUESTED** [review]

File: question/models/comment.py (hunk 2 above)
Comment (Karthikeyantestpress) [reply question/models/comment.py]: class QuestionComment(TimeStampedModel):
Renamed

File: question/models/comment.py (hunk 1 above)
Comment (Karthikeyantestpress) [reply question/models/comment.py]: class AnswerComment(TimeStampedModel): renamed

**Review by waranlogesh — APPROVED** [review]

**Pull request merged**